CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+
+# Contribution Rules📚:
+
+- You are allowed to make pull requests. We just merge it ;)
+- Do NOT remove other content.
+- Styling/code can be pretty, ugly or stupid, big or small as long as it works
```

Comment:
Sorry, but code has to follow some standards. Formatting and style of writing code has to be good. "as long as it works" is not acceptable. Can you please update.